### ISSUE TITLE
Submission must be set with metadata property 

### ISSUE THREAD

**Issue by bredar** [formio/formio.js#5724]
**Describe the bug**

The submission must include metadata property. This seems to be a new behaviour after upgrading from "formiojs" 4.20.

The code logic in Webform.js, seems to be wrong:

```javascript
 setValue(submission, flags = {}) {
        if (!submission || !submission.data) {
            submission = {
                data: {},
                metadata: submission.metadata,
            };
        }
```

**Version/Branch**
5.0.0-rc.69

**To Reproduce**

This does not work:
```this.formio.submission = { data: { test: "abc" } };```

This solves the problem
```this.formio.submission = { data: { test: "abc" }, metadata: {} };```

**Comment by brendanbond** [comment]
Hey @bredar thanks for reporting. We'll open an internal ticket

**Comment by daneformio** [comment]
Hi, @bredar , could you provide a JSfiddle that reproduces your issue? We tried to set it up in one here: https://jsfiddle.net/brendanbond/fhc80btz/ but I am not seeing any errors or anything.

**Comment by bredar** [comment]
I understand that you would prefer a reproduce but it seems not to be that easy. We are using angular formio. 
But I think it's quite obvious that the code above is not clean.

**Comment by lane-formio** [comment]
Resolved in formio/js@5.0.0-rc.72

**Comment by lane-formio** [comment]
<div>Agent Lane Doughtie linked Freshdesk ticket <a href="https://formhelp.freshdesk.com/a/tickets/47634">47634</a> for this issue.</div><br/>
